protecode/model.py

```diff
@@ -46,12 +46,15 @@ def name(self):
     def vulnerabilities(self) -> 'Iterable[Vulnerability]':
         return (Vulnerability(raw_dict=raw) for raw in self.raw.get('vulns'))
 
-    def highest_major_cve_severity(self) -> int:
+    def highest_major_cve_severity(self, ignore_triage:bool = False) -> int:
         try:
             return max(
                 map(
                     lambda v: v.cve_major_severity(),
-                    filter(lambda v: not v.historical(), self.vulnerabilities())
+                    [
```

Comment:
it get even harder to read like this. in particular, I do not think it is a good style to mix and nest filter and list comprehension.

consider introducing an inline function and pass it to the filter invocation.

alternatively, this might be a good place to use our FluentIterable